src/actions/fetchSinglePostAction.js

```diff
@@ -36,6 +50,7 @@ export default (slug) => async (dispatch) => {
 
     dispatch(fetchSuccess(post));
     dispatch(toggleLoading(false));
+    dispatch(await fetchRelatedPosts(post.type, slug));
```

Comment:
@dusmel Why are you awaiting this action?

Reply:
because otherwise, it will return a promise and dispatch take an object